docs/admin/daemons.md

```diff
@@ -92,6 +92,12 @@ when the pod is created, so it is ignored by the scheduler).  Therefore:
  - DaemonSet controller can make pods even when the scheduler has not been started, which can help cluster
    bootstrap.
 
+## How Daemon Pods work with taints and tolerations
+
+Daemon pods will not be deleted by `NodeController` in case of node problems, such as when node turns to
+`Unreachable` or `NotReady`. The reason is that the `NodeController` added an infinite tolerations of
```

Comment:
Thanks all for the comments, I was planning to make this as a reference to https://github.com/kubernetes/kubernetes.github.io/pull/2774 as there is a section explaining `daemonset` logic there.

Marking this as WIP to wait https://github.com/kubernetes/kubernetes.github.io/pull/2774 to be merged first before I update this PR.